assets/js/components/adminbar/AdminBarWidgets.stories.js

```diff
@@ -156,15 +161,43 @@ AnalyticsInactiveNew.storyName = 'Inactive: Analytics New CTA';
 AnalyticsInactiveNew.decorators = [
 	( Story ) => {
 		const setupRegistry = ( registry ) => {
-			// Set up the search console module store but provide no data.
+			provideModules( registry );
+
+			setupSearchConsoleMockReports( registry );
+		};
+
+		return (
+			<WithRegistrySetup func={ setupRegistry }>
+				<Story />
+			</WithRegistrySetup>
+		);
+	},
+];
+AnalyticsInactiveNew.parameters = {
+	features: [ 'zeroDataStates' ],
+};
+AnalyticsInactiveNew.scenario = {
```

Comment:
This issue doesn't call for added VRT scenarios, only Storybook. Let's hold on adding these as we haven't really established criteria for what should be covered or not for new scenarios.